sbin/wifibox

```diff
@@ -736,9 +743,18 @@ vm_stop() {
 	return 1
     fi
 
-    log info "Stopping guest ${WIFIBOX_VM}, managed by PID ${_pid}"
+    log debug "pid=${_pid}"
+
+    _gpid="$(${PS} -o pgid -p "${_pid}" | ${TAIL} -1)"
+
+    if [ -z "${_gpid}" ]; then
+	log warn "Guest is not running, hence not stopped"
+	return 1
+    fi
+
+    log info "Stopping guest ${WIFIBOX_VM}, managed by PID ${_pid}, GPID ${_gpid}"
 
-    if ! (${KILL} -SIGTERM "${_pid}" 2>&1 | capture_output debug kill); then
+    if ! (${KILL} -TERM "-${_gpid}" 2>&1 | capture_output debug kill); then
```

Comment:
I think the hyphen should not be made part of the quoting.  Otherwise the following will show up in the logs:

```
2024-09-11T07:56:38+0200 INFO  Stopping guest wifibox, managed by PID 418, GPID  418
2024-09-11T07:56:38+0200 DEBUG [kill] kill: illegal process id: - 418
```

This works for me and ShellCheck seems to like it too:

```diff
--- a/sbin/wifibox
+++ b/sbin/wifibox
@@ -754,7 +754,7 @@ vm_stop() {
 
     log info "Stopping guest ${WIFIBOX_VM}, managed by PID ${_pid}, GPID ${_gpid}"
 
-    if ! (${KILL} -TERM "-${_gpid}" 2>&1 | capture_output debug kill); then
+    if ! (${KILL} -TERM -"${_gpid}" 2>&1 | capture_output debug kill); then
        log warn "Guest could not be stopped gracefully"
     fi

```

Reply:
This implies a bug in the lines above -- it shouldn't matter where the - is. Let's fix that.